packages/cli/src/controllers/userSettings.controller.ts

```diff
@@ -0,0 +1,51 @@
+import { Patch, RestController } from '@/decorators';
+import { NpsSurveyRequest } from '@/requests';
+import { UserService } from '@/services/user.service';
+import { ApplicationError, type NpsSurveyState } from 'n8n-workflow';
+
+function getNpsSurveyState(state: unknown): NpsSurveyState | undefined {
```

Comment:
This is fine, but in the future we should define a schema and validate against that

Reply:
Ya it's not simple as a schema but covers all the cases without extra complexity